versions.tf

```diff
@@ -1,8 +1,9 @@
 terraform {
   backend "s3" {
-    bucket         = "jenkins-state-1988"
-    key            = "codepipeline-lambda"
-    region         = "us-east-1"
+    bucket  = "revature-terraform"
```

Comment:
As mentioned elsewhere, the terraform state is being kept in an S3 bucket that I already had pre-configured. I simply needed to update the configuration to reference my bucket.

It is also important to note, that this configuration had to have changed. It defaulted to a `jenkins-state-1988` S3 bucket, but I don't believe this is accessible to us. Therefore, we are required to use our own bucket.